displaygame.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
This is a little hacky 😄 (ie the os call from python for something we could do more elegantly from the python program itself). 

We can replace it with a window refresh using the `curses` package (part of the python standard lib):
- Official doc: https://docs.python.org/3/howto/curses.html
- And, as always, some kind of tutorial rehashing the official tutorial 😉  https://www.devdungeon.com/content/curses-programming-python

Reply:
Basically you just have to:
```python
import curses

# create a screen
scr = curses.initscr()

# Add the text to the screen buffer
scr.addstr(0, 0, "Example text added to coordinate (0, 0)")

# Display the result
scr.refresh()

```

Reply:
This has a little bit of overhead, but I think it's worth it since:

- `curses` is standard lib, so it's an investment in something stable for long-term use in any CLI you build
- In a terminal game you'll have a lot of increasingly complicated things to buffer to a screen, and it'll pay dividends to make a screen/window abstraction